Python/Python_news_reader/Python news reader.py

```diff
@@ -0,0 +1,44 @@
+'''
+This python program collects the top news headlines and their description from 'newsapi.org' website and will read the news headlines and
+description in his own voice.
+News API is a simple and easy-to-use API that returns JSON metadata for headlines and articles live all over the web right now.
+***Visit newsapi.org for documentation and for your API key***
+'''
+
+import time
+from win32com.client import Dispatch
+import requests
+
+def func():
+    url = (
+        'http://newsapi.org/v2/top-headlines?country=in&category=business&apiKey=paste your API key here')  # Only paste your API key here, don't change the whole link.
+    # Example: ' http://newsapi.org/v2/top-headlines?country=in&category=business&apiKey=9895kn34nah45kan87ha12HG34Ng3s6 '
+
+    response = requests.get(url)
+    m = response.json()
+    print(m)
+    lst = m['articles']
```

Comment:
The Variable names don't seem to follow any specification. Try running your Script through PyLint and make apt changes.